### PR TITLE
add events of this year to the 38c3 site

### PR DESCRIPTION
**Pull request by Simon-Laux**
because the event was still from last year.

### CODE REVIEW

File: 38c3.html [hunk 1]
```diff
@@ -161,23 +171,60 @@ <h2 id="test-delta-chat-without-registration">
...
+          Event on events.ccc.de</a
+        >
+      </p>
+    </div>
+    <div class="event">
+      <h4>
+        Chatmail-server for activists (real-life and upcoming usage)
+      </h4>
+      <div>19:00 - 20:30📍 Komona Table Space (hall H)</div>
+      <p>[description coming soon]</p>
```
Comment (Simon-Laux) [inline 38c3.html:224]: @missytake I don't know what it is exactly about, I guess best practices and the real-life threats vs theoretical ones?

File: 38c3.html [hunk 2]
```diff
@@ -161,23 +171,60 @@ <h2 id="test-delta-chat-without-registration">
...
       </p>
+    </div>
+    
+    <h3>🗓️ Day 3</h3>
+    <div class="event">
+      <h4>
+        Delta Chat 2025 roadmap and discussions
+      </h4>
+      <div>17:00 - 18:00📍 Komona Table Space (hall H)</div>
+      <p>Take part in shaping Delta Chats future, or just learn what the project might do in 2025</p>
```
Comment (Simon-Laux) [inline 38c3.html:210]: @hpk42 is this description ok?

Comment (hpk42) [reply 38c3.html:210]: good enough for a start.  let's merge and improve.

File: 38c3.html (hunk 1 above)
Comment (hpk42) [reply 38c3.html:224]: it's about setting up chatmail servers.

**Comment by hpk42** [comment]
i'd like to a PR on top, so i am merging this, as it corrects factual bad links and improves things.

**Pull request merged**